Backend/WebApi-PointMap/ErrorHandling/GeneralErrorHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿using DataAccessLayer.Database;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web;
+using static ServiceLayer.Services.ExceptionService;
+using System.ComponentModel.DataAnnotations;
+
+namespace WebApi_PointMap.ErrorHandling
+{
+    public class GeneralErrorHandler
+    {
+        public static HttpResponseMessage HandleException(Exception e)
+        {
```

Comment:
we dont have this anymore

Reply:
Yup the physical files wasn't removed in my pull, but it was removed from the project